src/cavendish_particle_tracks/_analysis.py

```diff
@@ -19,7 +19,14 @@
     "A": [-15.00, 8.68],
 }  # cm
 
-EXPECTED_PARTICLES = ["New particle", "Σ+", "Σ-", "Λ0"]
+EXPECTED_PARTICLES = [
+    "New particle",
+    "Σ⁺ ⇨ p + π⁰",
```

Comment:
Is there a way to define these symbols to be readable by Excel (#58)?

Reply:
Yea it likely has to do with how different language sets are handled. Easiest might be writing the characters in excel and copying them back into the code. Whatever version of the character set Excel uses is bound to be compatible.

Reply:
Looking a bit into this, I am able to change the encoding used when importing into excel to UTF8, in which case the symbols are correct. Issue is that the default encoding in Excel, [seems to be machine specific](https://stackoverflow.com/questions/508558/what-charset-does-microsoft-excel-use-when-saving-files#:~:text=From%20memory%2C%20Excel%20uses%20the%20machine%2Dspecific%20ANSI%20encoding.) (in my case `Macintosh`).

We could change it to whatever default is in the lab machines, but there is no guarantee that it would work for other machines the user might use for the data analysis. 

I suggest we leave the cvs file to be UTF8 encoded
https://github.com/palvarezc/cavendish-particle-tracks/blob/6e6a2bc1f7c20cca4b723ebb375d6eec7fc7bcf1/src/cavendish_particle_tracks/_widget.py#L384
and add a note about how to correctly import UTF8-encoded files in Excel to the lab manual.

Reply:
Reading through the same here. Seems it isn't simple to change the default behaviour as well at first glance. Happy to keep it that way.